test/users.test.js

```diff
@@ -3,10 +3,23 @@ const request = require('supertest');
 const app = require('../app');
 const db = require('../app/models');
 // const { factoryByModel } = require('./factory/factory_by_models');
-const { createUser, buildUser } = require('./factory/users');
+const { createUser, buildUser, createMany } = require('./factory/users');
 
 let user = '';
 
+const loginNewUser = async loginUser => {
+  await request(app)
+    .post('/users')
+    .send(loginUser.dataValues);
+  const { body } = await request(app)
+    .post('/users/sessions')
+    .send({
+      email: loginUser.dataValues.email,
+      password: loginUser.dataValues.password
+    });
+  return body;
+};
+
 beforeEach(async () => {
   await db.User.destroy({ truncate: { cascade: true } });
   user = await buildUser();
```

Comment:
If you want to only use the properties here, you can use the `factory.attrs()` funtion